scripts/create_test_subset.py

```diff
@@ -146,8 +181,21 @@ def main(
         samples = [s for s in all_samples if s['id'] in sample_set]
 
     else:
-        assert samples_n
-        samples = random.sample(all_samples, samples_n)
+        if _additional_families:
+            _additional_samples.extend(
+                get_samples_for_families(project, _additional_families)
+            )
+
+        samples_to_subset = [
+            sample
+            for sample in all_samples
+            if sample['id'] not in set(_additional_samples)
```

Comment:
Don't cast to a set inside the loop, can you bring this (and the other examples) out of the loop please.